docker/builder/Dockerfile

```diff
@@ -0,0 +1,55 @@
+# Run interactively with docker-compose run builder /bin/bash
+
+FROM r-base:3.6.2
+
+# switch to bash from dash
+RUN ln -s bash /bin/sh.bash
+RUN mv /bin/sh.bash /bin/sh
+
+# add repositories to debian environment
+RUN echo "deb http://deb.debian.org/debian stretch main" >> /etc/apt/sources.list
+RUN echo "deb-src http://deb.debian.org/debian stretch main" >> /etc/apt/sources.list
+RUN echo "deb http://deb.debian.org/debian-security/ stretch/updates main" >> /etc/apt/sources.list
+RUN echo "deb-src http://deb.debian.org/debian-security/ stretch/updates main" >> /etc/apt/sources.list
+RUN echo "deb http://deb.debian.org/debian stretch-updates main" >> /etc/apt/sources.list
+RUN echo "deb-src http://deb.debian.org/debian stretch-updates main" >> /etc/apt/sources.list
+
+# install system libraries
+RUN apt-get update; apt-get -y install python-pip-whl=9.0.1-2+deb9u1 \
+    python-pip \
+    curl \
+	pkg-config \
+	libxml2-dev \
+	libcurl4-gnutls-dev \
+	libssl-dev \
+	librasqal3-dev \
+	libraptor2-dev \
+	librdf0-dev \
+	libv8-dev \
+	dos2unix \
+	git \
+	build-essential \
+	ca-certificates \
+	gcc \
+	libpq-dev \
+	make \
+	python-pip \
+	python2.7 \
+	python2.7-dev \
+	ssh \
+	default-libmysqlclient-dev \
+	&& apt-get -y autoremove \
+    && apt-get clean
+	
+# load R packages
+RUN Rscript -e 'install.packages(c("config", "httr", "igraph", "SPARQL", "rdflib", "tibble", "uuid", "config"))'
```

Comment:
* We should note the current working versions of all the packages that are explicitly being installed here in the documentation (docs currently only have "config", "httr", "igraph", "SPARQL").  This install command will always pull the latest versions, we should have a reference in case a future package change breaks the current scripts.  

  If necessary to install older versions in the future, it seems like specific R package versions can be installed using [install.versions()](https://www.rdocumentation.org/packages/versions/versions/0.3/topics/install.versions) instead if install.packages() (see [SO](https://stackoverflow.com/questions/17082341/installing-older-version-of-r-package)).

* The R package install takes a long time, install.packages has a flag to use [multiple CPUs](https://www.jumpingrivers.com/blog/speeding-up-package-installation/) which may speed this up.  This does not need to be changed, just a note if this is revisited.